platforms/jvm/testing-jvm/src/test/groovy/org/gradle/api/tasks/testing/testng/TestNGOptionsTest.groovy

```diff
@@ -103,6 +105,7 @@ class TestNGOptionsTest extends Specification {
             setListeners([prefix + "Listener"] as Set)
             setParallel(prefix + "Parallel")
             setThreadCount(intValue)
+            setSuiteThreadPoolSize(intValue)
```

Comment:
❗ As the `intValue` here is the same as the default value, setting to 1 isn't a good test.  Please add a test that sets it to something different and verifies the result.

Reply:
I agree, 1 is not suitable. What if I change intValue to, say, 5? It's a very lazy solution, but it meets the needs of this test without complicating it by adding another variable/another test. Let me know If this is too cheeky -  I'll try to add a another test)

Reply:
I think that would be fine.